builder/dockerfile/parser/testfiles/health/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM debian
+ADD check.sh main.sh /app/
+CMD /app/main.sh
+HEALTHCHECK
+HEALTHCHECK --interval=5s --timeout=3s --retries=1 --grace=4s --exit-on-unhealthy \
```

Comment:
This is a unit-test for the parser. It's not intended as an example for users (it also specifies mutliple healthchecks and other unreasonable things).